### DISCUSSION TITLE
Enable multi-thread in command line

### DISCUSSION THREAD

**Discussion by gibhsu** [frePPLe/frepple#466] [Q&A]
Can we enable the solver multi-thread in the command line and any other way?

Below is an example I got from FrePPLE cloud.
--constraint=1 --plantype=1 --env=supply --background

**Comment by jdetaeye** [comment]
There is no need for a command line option for this. It's fully automatic.   
The algorithm detects independent subproblems, and automatically solves these in parallel threads.

Note: the plan.loglevel needs to be 0 to allow this parallelization. We do this to avoid messing up the output in the log file.
